zha/zigbee/cluster_handlers/measurement.py

```diff
@@ -19,6 +19,7 @@
     RelativeHumidity,
     SoilMoisture,
     TemperatureMeasurement,
+    WindSpeedMeasurement,
```

Comment:
The naming in zigpy is inconsistent, the cluster doesn't have a `Measurement` suffix:

```suggestion
    WindSpeed,
```